vue/src/vueTextMask.js

```diff
@@ -1,24 +1,38 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
 
 export default {
-  textMaskInputElement: null,
 
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
-  },
+  bind (el, binding, vnode) {
+    // Check for empty value is good for list rendering (some inputs has no masks)
+    if (!binding.value) {
+      return
+    }
+
+    binding.value.inputElement = el
+
+    const textMaskInputElement = createTextMaskInputElement(binding.value)
```

Comment:
can this be changed to `this.textMaskInputElement = createTextMaskInputElement(binding.value)`